Wikipedia/Code/ArticleViewController+Announcements.swift

```diff
@@ -132,7 +132,14 @@ extension ArticleViewController {
             return false
         }
 
-        guard let yirDataController = try? WMFYearInReviewDataController() else {
+        var userId: Int?
+
+        if let siteURL = dataStore.languageLinkController.appLanguage?.siteURL,
+           let userID = dataStore.authenticationManager.permanentUser(siteURL: siteURL)?.userID {
+            userId = userID
+        }
```

Comment:
(See comment above) You will be able to delete this `userId` setup code if you pass in `userID` as a parameter to `populateYearInReviewReportData`.